### PR TITLE
Fixes #13314: Update agent dependencies

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/13314

### CODE REVIEW

**Comment by ncharles** [comment]
Fusion will be updated in another ticket ?

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

File: rudder-inventory-ldap/SOURCES/patches/openldap/0002-dn2id-modrdn-idlcache-sub.patch [hunk 1]
```diff
@@ -1,47 +0,0 @@
-commit 6373e9f0ba522e209b53e468476ca936c12094f8
```
Comment (amousset) [inline rudder-inventory-ldap/SOURCES/patches/openldap/0002-dn2id-modrdn-idlcache-sub.patch]: We do build bdb backend anymore.

**Pull request merged**